yt_dlp/extractor/tempo.py

```diff
@@ -24,30 +24,16 @@ def _real_extract(self, url):
         display_id = self._match_id(url)
         webpage = self._download_webpage(url, display_id)
 
-        player_key, widget_id = self._search_regex(
-            r'<ivs-player\s*[^>]+data-ivs-key\s*=\s*"(?P<player_key>[\w]+)[^>]+\bdata-ivs-wid="(?P<widget_id>[\w-]+)',
-            webpage, 'player_key, widget_id', group=('player_key', 'widget_id'))
+        player_key, video_id = self._search_regex(
+            r'<ivs-player\s*[^>]+data-ivs-key\s*=\s*"(?P<player_key>[\w]+)\s*[^>]+\bdata-ivs-vid="(?P<video_id>[\w-]+)',
+            webpage, 'player_key, video_id', group=('player_key', 'video_id'))
 
         json_ld_data = self._search_json_ld(webpage, display_id)
 
-        json_data = self._download_json(
-            f'https://ivxplayer.ivideosmart.com/prod/widget/{widget_id}',
-            display_id, query={'key': player_key})
-        formats, subtitles = self._extract_m3u8_formats_and_subtitles(
-            json_data['player']['video_url'], display_id, ext='mp4')
-
-        return {
-            'id': str(json_data['ivx']['id']),
-            'display_id': display_id,
-            'formats': formats,
-            'subtitles': subtitles,
-            'title': (self._html_search_meta('twitter:title', webpage) or self._og_search_title(webpage)
-                      or traverse_obj(json_data, ('ivx', 'name'))),
-            'duration': int_or_none(traverse_obj(json_data, ('ivx', 'duration'))),
-            'thumbnail': (self._html_search_meta('twitter:image:src', webpage) or self._og_search_thumbnail(webpage)
-                          or traverse_obj(json_data, ('ivx', 'thumbnail_url'))),
-            'description': (json_ld_data.get('description') or self._html_search_meta(['description', 'twitter:description'], webpage)
-                            or self._og_search_description(webpage)),
-            'timestamp': parse_iso8601(traverse_obj(json_data, ('ivx', 'created_at'))),
-            'tags': str_or_none(self._html_search_meta('keywords', webpage), '').split(','),
-        }
+        return self.url_result(
+            f'ivxplayer:{video_id}:{player_key}', display_id=display_id,
+            thumbnail=self._html_search_meta('twitter:image:src', webpage) or self._og_search_thumbnail(webpage),
+            tags=try_call(lambda: self._html_search_meta('keywords', webpage)).split(','),
```

Comment:
```suggestion
            tags=try_call(lambda: self._html_search_meta('keywords', webpage).split(',')),
```
otherwise `None.split` can raise